model/Check/System/AwsRDSFreeSpaceCheck.php

```diff
@@ -69,23 +56,27 @@ protected function doCheck(): Report
 
         if ($freeSpacePercentage < 30) {
             $this->logError(__('Free space on RDS storage') . '< 30%');
-            $report = new Report(Report::TYPE_ERROR, round($freeSpacePercentage).'%');
+            $report = new Report(Report::TYPE_ERROR, round($freeSpacePercentage) . '%');
         } elseif ($freeSpacePercentage < 50) {
-            $report = new Report(Report::TYPE_WARNING, round($freeSpacePercentage).'%');
+            $report = new Report(Report::TYPE_WARNING, round($freeSpacePercentage) . '%');
         } else {
-            $report = new Report(Report::TYPE_SUCCESS, round($freeSpacePercentage).'%');
+            $report = new Report(Report::TYPE_SUCCESS, round($freeSpacePercentage) . '%');
         }
+
         $report->setData([self::PARAM_VALUE => round($freeSpacePercentage)]);
 
         return $report;
     }
 
     /**
      * @return bool
+     * @throws SystemCheckException
      */
     public function isActive(): bool
     {
-        return $this->isAws();
+        $instanceData = $this->getInstanceData($this->getStackId($this->getRDSHost()));
```

Comment:
**question**: How we can get RDS instance data in a non-AWS environment? I'm asking as we first pull instance data and only then check if it is AWS or not.

Reply:
fixed